aequilibrae/utils/python_signal.py

```diff
@@ -14,7 +14,7 @@
     else:
         from tqdm import tqdm  # type: ignore
 
-show_status = os.environ.get("AEQ_SHOW_PROGRESS", "TRUE") == "TRUE"
+show_status = os.environ.get("AEQ_SHOW_PROGRESS", "TRUE") == "TRUE" or missing_tqdm
```

Comment:
If we're missing tqdm then surely we can't show a progress bar? Or is this for QGIS?
```suggestion
show_status = os.environ.get("AEQ_SHOW_PROGRESS", "TRUE") == "TRUE" and not missing_tqdm
```